programs/disks/DisksApp.cpp

```diff
@@ -171,6 +162,20 @@ int DisksApp::main(const std::vector<String> & /*args*/)
         throw Exception(ErrorCodes::BAD_ARGUMENTS, "No config-file specifiged");
     }
 
+    if (config().has("save-logs"))
+    {
+        auto log_level = config().getString("log-level", "trace");
+        Poco::Logger::root().setLevel(Poco::Logger::parseLevel(log_level));
+
+        auto log_path = config().getString("logger.clickhouse-disks", "/var/log/clickhouse-server/clickhouse-disks.log");
+        Poco::Logger::root().setChannel(Poco::AutoPtr<Poco::FileChannel>(new Poco::FileChannel(log_path)));
+    }
+    else
+    {
+        auto log_level = config().getString("log-level", "none");
```

Comment:
Shouldn't we have default value for `log-level` which will be used if neither `save-logs` nor `log-level` options are provided?

Reply:
`config().getString("log-level", "none")` will return "none" if `log-level` is not set

Reply:
Yes, understand. But, imo, it's a strange place to set a default value for `log-level`. A proper one would be to provide a default value when we add options, see my suggestion on line 61. I guess this `else` is not necessary. We'll just execute applying log level unconditionally:
```
Poco::Logger::root().setLevel(Poco::Logger::parseLevel(log_level));
```

Reply:
The logic behind that was that if we do not set `save-logs` and `log-level` parameters, then the default value is `none`, and no logs are printed anywhere. If we want to print some logs to the terminal we can just set `log-level`.

But if we set the `save-logs` parameter and nothing else, then it would be strange to still have `none` level by default, so we set `trace` level by default.

https://github.com/ClickHouse/ClickHouse/blob/d7d0794d2f57e27a2b3eef3a3889ef272ee95338/programs/disks/DisksApp.cpp#L165-L167

Reply:
Well, if from usability POV having 2 different `log-level` default values is better, then ok